VSharp.SILI.Core/Pointers.fs

```diff
@@ -253,7 +272,7 @@ module internal Pointers =
         simplifyBinaryOperation mtd OperationType.Subtract State.empty x y (typeof<System.Void>.MakePointerType()) fst
 
     let isPointerOperation op t1 t2 =
-        let isNearlyPtr t = Types.isPointer t || Types.isReference t
+        let isNearlyPtr t = Types.isReference t || Types.isPointer t
```

Comment:
Strange change. Maybe `isNearlyPtr` should be totally eliminated: replace it with just `Types.isPointer`